jsonschema/README.md

```diff
@@ -0,0 +1,17 @@
+# JSON SCHEMA
+
+This folder contains the JSON Schema of execution event, and this schema will
+be used by [cloudevent publisher](https://github.com/flyteorg/flyteadmin/blob/a0ca4b07d3b1c3cccfe3830307df50bc73152ddb/pkg/async/cloudevent/implementations/cloudevent_publisher.go#L96).
+
+The URL for the JSON file will be included with the cloudevent message. For example
+```json
+{
+    "specversion" : "1.0",
+    "type" : "com.flyte.resource.workflow",
```

Comment:
what is the corresponding type for node and task execution events?

Reply:
In our last discussion, we decided that we would only support workflow execution events for now, so I only added cloudevent publisher to `execution_manager.go` 
https://github.com/flyteorg/flyteadmin/pull/355/files#diff-9b311b663f43eccf79220bfcae3a565ad26d83532c945f39cfc1f7ba029c22e3R98-R100

Reply:
very java